src/components/core/MultipleSelect.tsx

```diff
@@ -0,0 +1,284 @@
+  const element = {
+    display: 'flex',
+    width: 'fit-content',
+    alignItems: 'center',
+    margin: '0 10px 0 0',
+    borderRadius: `${theme.shape.spacing(5)}px`,
+    flexShrink: 0,
+    overflowX: 'auto',
+    transition: 'background-color 0.25s',
+    textTransform: 'uppercase',
+  };
+
+  return {
+    groupContainer: {
+      position: 'relative',
+      marginTop: -3,
+    },
+    label: {
+      width: '100%',
+      position: 'relative',
+      fontSize: '0.9vw',
+      whiteSpace: 'nowrap',
+      color: theme.palette.neutral,
+    },
+    selectContainer: {
+      position: 'relative',
+      width: `${theme.shape.spacing(92)}px`,
+      height: `${theme.shape.spacing(6)}px`,
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      boxSizing: 'border-box',
+      padding: `${theme.shape.spacing(1)}px`,
+      overflowX: 'auto',
+      overflowY: 'hidden',
+      backgroundColor: `${theme.palette.background}`,
+      color: `${theme.palette.neutral}`,
+      transition: 'border-color 0.3s, box-shadow 0.3s',
+      scrollbarColor: `${theme.palette.primary} ${theme.palette.background}`,
+      scrollbarWidth: 'thin',
+      border: `1px solid ${theme.palette.neutralMedium}`,
+      borderRadius: `${theme.shape.spacing(5)}px`,
+      '&:hover, &:active ': {
+        boxShadow: '0 0 0 0.2rem #007bff25',
+        borderColor: `${theme.palette.primary}`,
+        cursor: 'pointer',
+      },
+      '&::-webkit-scrollbar': {
+        height: 6,
+        marginLeft: `${theme.shape.spacing(2)}px`,
+      },
+      /* Track */
+      '&::-webkit-scrollbar-track': {
+        boxShadow: 'inset 0 0 5px grey',
+        backgroundColor: theme.palette.background,
+      },
+
+      /* Handle */
+      '&::-webkit-scrollbar-thumb': {
+        background: `${theme.palette.primary}`,
+      },
+      /* Handle on hover */
+      '&::-webkit-scrollbar-thumb:hover': {
+        background: `${theme.palette.positive}`,
+      },
+      '&::-webkit-scrollbar-button': {
+        backgroundColor: 'transparent',
+      },
+    },
+    elements: {
+      display: 'flex',
+      boxSizing: 'border-box',
+    },
+    selectedElement: {
+      ...element,
+      padding: '11px 16px 11px 24px',
+      backgroundColor: `${theme.palette.primary}`,
+      lineHeight: '18px',
+      '&:hover': {
+        cursor: 'default',
+        backgroundColor: `${theme.palette.positive}`,
+      },
+      '& > span': {
+        fontWeight: 600,
+      },
+    },
+    removeIcon: {
+      flexShrink: 0,
+      color: `${theme.palette.neutral}`,
+      marginLeft: `${theme.shape.spacing(1)}px`,
+      '&:hover': {
+        color: `${theme.palette.negative}`,
+        cursor: 'pointer',
+      },
+    },
+    suggestionsTitle: {
+      marginTop: `${theme.shape.spacing(3)}px`,
+      marginBottom: `${theme.shape.spacing(2)}px`,
+      textTransform: 'uppercase',
+    },
+    suggestions: {
+      width: `${theme.shape.spacing(92)}px`,
```

Comment:
Would avoid fixing the width of a component like that. Would suggest going for 100%, flex-grow or something similar.
Another idea is to incorporate media queries and have it responsive